fast_llm/layers/transformer/config.py

```diff
@@ -127,6 +127,11 @@ class TransformerArchitectureConfig(BaseModelArchitectureConfig):
         desc="Scale for the rotary positional embeddings. Default: -math.log(10000) = -9.210",
         hint=FieldHint.feature,
     )
+    rotary_scaling_type: RotaryScalingType = Field(
```

Comment:
how about `rotary_embedding_type` so we make it more general?
Or better, merge with `use_rotary_embeddings` into `rotary_embeddings: RotaryEmbeddingType | None`

Reply:
Would make sense.
Although that would make this a breaking change right? Would that be OK?

Reply:
@tscholak @jlamypoirier I could merge into `rotary_embeddings: RotaryEmbeddingType | None`, and keep `use_rotary_embeddings` as a deprecated argument to maintain backwards compatibility. wdyt?

Reply:
ok!

Reply:
Hmm, in addition to breaking existing checkpoints it will make conversion more complex than needed. Let's postpone to another PR?

Reply:
Sounds good! We only want to unblock model conversion at this point. Thanks!